base/dates/io.jl

```diff
@@ -380,10 +383,12 @@ end
 # Standard formats
 const ISODateTimeFormat = DateFormat("yyyy-mm-dd\\THH:MM:SS.s")
 const ISODateFormat = DateFormat("yyyy-mm-dd")
+const ISOTimeFormat = DateFormat("HH:MM:SS.s")
```

Comment:
Confirmed to be an ISO8601 standard: https://en.wikipedia.org/wiki/ISO_8601#Times